src/components/dashboard/FaceRecognition/FRError.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import { Image, View, Platform } from 'react-native'
+import { Image, Platform, View } from 'react-native'
```

Comment:
1. as .svg graphics is just an small text file it could be bundled to the JS code like it's done in CRA:
https://create-react-app.dev/docs/adding-images-fonts-and-files

I think the corresponding solution could be found for RN & RN-Web

2. Use destructiring to simplify access to nested props :

```
// firstly, decompose props object
const FRError = ({ styles, screenProps }) => {
 // then decompose screenProps
 const { screenState | = screenProps;
 let { isValid, error: reason } = screenState;

 // ...
}
```


3. No side effects should be performed during render.
  - pop isValid to the screenProps in the effect runs every isValid changes:

```
useEffect(() => screenProps.pop({ isValid }), [isValid]);
```
- any logging should be performed in the effects / handlers too
- process `reason` variable and calculate error message and title inside effect runs each `reason` changes
- use memoization. wrap getting fullName to useMemo, wrap gotoFR to useCallback:
```
const fullName = useMemo(() => {
  const store = GDStore.useStore()
  const { fullName } = store.get('profile')

  return fullName;
}, []);

const gotoFR = useCallback(() => {
   screenProps.navigateTo('FaceVerification', { showHelper: true })
}, [screenProps]);
```

Reply:
done